### PR TITLE
Set extra <meta> fields in HTML <head>

### PR DESCRIPTION
**Pull request by dhimmel**
Closes https://github.com/manubot/rootstock/issues/176
Closes https://github.com/manubot/rootstock/issues/267

- [x]  update with `manubot` commit hash once https://github.com/manubot/manubot/pull/138 and https://github.com/manubot/manubot/pull/169 have been merged.

### CODE REVIEW

**Comment by dhimmel** [comment]
Tested https://github.com/manubot/rootstock/pull/277/commits/2487ffe315a4769c5bc598cf77f4aecbf5b9453d locally and can confirm that `manuscript.html` contains the added fields to the HTML `<head>` while preserving other `header-includes` like #275.

**Comment by dhimmel** [comment]
helpful references:

```html
<!--
https://github.com/jgm/pandoc-templates/blob/master/default.html5
https://github.com/manubot/rootstock/issues/176
https://github.com/manubot/rootstock/issues/203
https://scholar.google.com/intl/en/scholar/inclusion.html#indexing
https://larsgw.blogspot.com/2018/07/journal-metadata-authors-institutions.html
https://ogp.me
https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup.html
-->
```

File: USAGE.md [hunk 1]
```diff
@@ -191,6 +191,28 @@ funders: GBMF4552  # optional
...
 Note that `affiliations` should be a list to allow for multiple affiliations per author.
 
+### Thumbnail
+
+A thumbnail is an image used to visually represent the manuscript,
+such as when a manuscript is shared on social media or added to the [Manubot catalog](https://manubot.org/catalog/).
+Specify a thumbnail in any of the following ways:
+
+1. placing an image named `thumbnail.png` anywhere in the manuscript repository (for example, in the root directory).
+2. setting `thumbnail` in `metadata.yaml` to a repository-relative path where the image file is located.
```
Comment (dhimmel) [inline USAGE.md]:
```suggestion
2. setting `thumbnail` in `metadata.yaml` to a path, relative to the repository root, where the image file is located.
```

File: USAGE.md [hunk 2]
```diff
@@ -191,6 +191,28 @@ funders: GBMF4552  # optional
...
+    Example:
+    ```yaml
+    thumbnail: build/assets/thumbnail-1000x1000.png
+    ```
+3. setting `thumbnail` in `metadata.yaml` to a URL where the image is located.
```
Comment (dhimmel) [inline USAGE.md]:
```suggestion
3. setting `thumbnail` in `metadata.yaml` to an absolute URL where the image is located.
```

**Comment by dhimmel** [comment]
There is a problem where our header-includes is inserted in the `<body>` rather than the `<head>`.

**Comment by dhimmel** [comment]
I am going to merge this and start updating manuscripts I control to test it in the wild.

**Comment by dhimmel** [comment]
I am going to merge this and start updating manuscripts I control to test it in the wild.

**Comment by agitter** [comment]
This update may have affected how Google Scholar recognizes the HTML version of Manubot manuscripts.  Deep review's 2019 update was added as a new work:
image

Has anyone else noticed Manubot manuscripts being detected by Google Scholar after the new `<meta>` fields were added?

**Comment by dhimmel** [comment]
> Deep review's 2019 update was added as a new work

Interesting that the 2019 update is now detected as a new manuscript. If this resulted from the metadata updates, I'd suspect the canonical link is involved. Here's the bibtex scholar exports for that citation:

```bib
@techreport{ching2019opportunities,
  title={Opportunities and obstacles for deep learning in biology and medicine: 2019 update},
  author={Ching, Travers and Himmelstein, Daniel S and Beaulieu-Jones, Brett K and Kalinin, Alexandr A and Do, Brian T and Way, Gregory P and Ferrero, Enrico and Agapow, Paul-Michael and Zietz, Michael and Hoffman, Michael M and others},
  year={2019},
  institution={Manubot}
}
```

Not very comprehensive.

**Pull request merged**